src/ingest_validation_tools/enums.py

```diff
@@ -87,3 +67,27 @@
         'phosphopeptides'
     ]
 }
+
+def _test_assay_name_compatibility():
```

Comment:
This function is a no-op and is never called... were the top-level doc tests not being called, or was there some other problem?

Reply:
The top-level doctest was not being called; I had to bind it into a function to get 'pytest --doctest-modules' to find it.  It may be that your change to the TODO block in the comment someone reactivated the doctest in your corresponding PR, but I don't understand the mechanism that would have done that.